packaging/deb/postinst

```diff
@@ -14,11 +15,35 @@ if [ "$1" = "configure" ]; then
             /usr/bin/bee init --config /etc/bee/bee.yaml >/dev/null 2>&1
             chown -R bee:bee /var/lib/bee/keys
         fi
-        if [ -f /var/lib/bee/keys/swarm.key ]; then
-            parse_json() { echo $1|sed -e 's/[{}]/''/g'|sed -e 's/", "/'\",\"'/g'|sed -e 's/" ,"/'\",\"'/g'|sed -e 's/" , "/'\",\"'/g'|sed -e 's/","/'\"---SEPERATOR---\"'/g'|awk -F=':' -v RS='---SEPERATOR---' "\$1~/\"$2\"/ {print}"|sed -e "s/\"$2\"://"|tr -d "\n\t"|sed -e 's/\\"/"/g'|sed -e 's/\\\\/\\/g'|sed -e 's/^[ \t]*//g'|sed -e 's/^"//' -e 's/"$//' ; }
-            echo "Please make sure there is sufficient eth and bzz available on $(parse_json $(cat /var/lib/bee/keys/swarm.key) address)"
-            echo "You can get both goerli eth and goerli bzz from https://faucet.ethswarm.org"
+        db_input high bee/clef-enable || true
+        if db_go; then
+            db_get bee/clef-enable
+            if [ "$RET" = true ]; then
+                grep -v BEE_CLEF_SIGNER_ENABLE /etc/default/bee > /etc/default/bee.tmp
+                echo "BEE_CLEF_SIGNER_ENABLE=true" >> /etc/default/bee.tmp && mv /etc/default/bee.tmp /etc/default/bee
+            fi
         fi
+        set +e
+        RESP=$(BEE_CLEF_SIGNER_ENABLE=$RET /usr/bin/bee init --config /etc/bee/bee.yaml 2>&1)
+        set -e
+        case "$RESP" in
+            Error*)
+                START=false
+                echo "Enabled clef-signer but clef is not running."
+                echo "Check https://docs.ethswarm.org/ for more info and how to fix."
+                echo "Or install latest release of bee-clef from https://github.com/ethersphere/bee-clef and reinstall bee."
+                echo "Start bee with systemctl --no-reload start bee.service"
+                ;;
+            *)
+                ETH_ADDRESS=$(echo "$RESP" | grep ethereum | cut -d' ' -f6 | tr -d '"')
```

Comment:
We should make output of init command standarized for parsing with an optional flag instead to relay on cut fields of the log message. Someone may change the log message and break this requirement.